olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/PartitionsRequest.h

```diff
@@ -218,11 +186,11 @@ class DATASERVICE_READ_API PartitionsRequest final {
    *
    * @return A string representation of the request.
    */
-  std::string CreateKey(const std::string& layer_id) const {
+  std::string CreateKey(const std::string& layer_id, boost::optional<int64_t> version) const {
```

Comment:
This is an API break. Please provide a default value for the second parameter. Then the users are not forced to change their code.